### PR TITLE
Update util.py

### PR DESCRIPTION
**Pull request by mushroom2**
initialization on osx quick bug fix

### CODE REVIEW

File: fulltext/util.py [hunk 1]
```diff
@@ -135,7 +135,10 @@ def warn(msg):
 
 def is_windows():
     """True if the platform is Windows."""
-    return 'win' in sys.platform
+    if sys.platform != 'darwin':  # aplle osx is not windows
+        return 'win' in sys.platform
+    else:
+        return False
```
Comment (giampaolo) [inline fulltext/util.py:141]: I think this should just be `return os.name == 'nt'` (this is what I do in psutil)

Comment (mushroom2) [reply fulltext/util.py:141]: and what will be next steps?

Comment (btimby) [reply fulltext/util.py:141]: @mushroom2 Can you make the change as Giampaolo suggests. If you then open a new PR I will merge it.

Comment (mushroom2) [reply fulltext/util.py:141]: ok

**Pull request closed without merging**